sm_value_store/src/VerboseValueStore.cpp

```diff
@@ -74,5 +80,32 @@ std::vector<KeyValueStorePair> VerboseValueStore::getChildren() const {
   return verboseChildren;
 }
 
+ValueHandle<bool> VerboseExtendibleValueStore::addBool(const std::string & path, bool initialValue) {
+  return logValue(__func__, path, vs_->addBool(path, initialValue)/*, initialValue*/);
+}
+ValueHandle<int> VerboseExtendibleValueStore::addInt(const std::string & path, int initialValue) {
+  return logValue(__func__, path, vs_->addInt(path, initialValue)/*, initialValue*/);
+}
+ValueHandle<double> VerboseExtendibleValueStore::addDouble(const std::string & path, double initialValue) {
+  return logValue(__func__, path, vs_->addDouble(path, initialValue)/*, initialValue*/);
+}
+ValueHandle<std::string> VerboseExtendibleValueStore::addString(const std::string & path, std::string initialValue) {
+  return logValue(__func__, path, vs_->addString(path, initialValue)/*, initialValue*/);
+}
+
+ExtendibleKeyValueStorePair VerboseExtendibleValueStore::getExtendibleChild(const std::string & key) const {
+  log_(std::string(__func__) + "(" + key + ")");
+  auto log = log_;
+  return ExtendibleKeyValueStorePair(key, std::make_shared<VerboseExtendibleValueStore>(vs_->getExtendibleChild(key), [log, key](const std::string &text){ log("extendible child(" + key + "):" + text);}));
```

Comment:
I suspect `extendible child` to be too long in many cases. What about `EC` instead?